### PR TITLE
Fix/changing from overflow to font case [Delivers #103897646]

### PR DESCRIPTION
**Pull request by chiefGui**
@jonschreiber @fosterdill 

Before:

before

After:

after

### CODE REVIEW

**Comment by evandrodutra** [comment]
:+1:

File: src/components/live-displays/Grid/GridAsset.scss [hunk 1]
```diff
@@ -90,10 +90,7 @@ $column2-left-position: $asset--normal-width * 2;
 }
 
 
-.grid-captionated-asset .grid-tweet {
-  width: 50%;
-  overflow: scroll;
-}
```
Comment (fosterdill) [inline src/components/live-displays/Grid/GridAsset.scss]: removing this overflow: scroll, won't that just go back to the old problem of having the text go outside the box or get cut off?

Comment (chiefGui) [inline src/components/live-displays/Grid/GridAsset.scss]: I added the default font of the display wall as Georgia to get away of that problem, according to the initial specs—which was a mistake of mine not doing this before.

**Comment by jonschreiber** [comment]
:-1: changing the font won't fix the problem with long tweets or long instagram captions. We need that overflow: scroll

**Comment by chiefGui** [comment]
Actually, @jonschreiber, Artur designed that display thinking in tweets with 140 characters—their maximum allowed. The problem was that typography wasn't corresponding to Artur's designed expectation—as I said to Dylan, it was my fault for not setting the default font as Georgia since the beginning.

In other words, the old font had different dimensions compared to Georgia. As you can see in the preview I used as an example, the tweet tile didn't break there and it was something with 140 characters.

If we want something more precise, we need to talk to Artur to rethink the tweet tile design.

**Pull request closed without merging**